dhall-docs/src/Dhall/Docs.hs

```diff
@@ -92,18 +116,18 @@ parserInfoOptions =
         <>  Options.Applicative.progDesc progDesc
         )
 
-{-| Fetches a list of all dhall files in a directory. This is not the same
-    as finding all files that ends in @.dhall@, but finds all files that
-    successfully parses as a valid dhall file.
+{-| Fetches a list of all dhall files in a directory along with its `Header`.
+    This is not the same as finding all files that ends in @.dhall@,
+    but finds all files that successfully parses as a valid dhall file.
```

Comment:
Not related to this PR, but it seems like this functions returns a `.git/HEAD` file (containing something like `ref: refs/heads/master`). It seems like `exprAndHeaderFromText` is not enough as the `HEAD` file is syntactically valid.

Reply:
Wow good noticed. I thought that exprAndHeaderFromText also does typechecking. A simple solution could be to typecheck the expression as well.

Reply:
> A simple solution could be to typecheck the expression as well.

That will require import resolution too, so it might be a bit of work.

I believe we'll eventually need that anyway, though.

Let's discuss this in our chat today.

Reply:
Sure

Reply:
That’s another clue that we should do the simple-stupid thing instead, namely only go by the file ending.

Reply:
I submitted a PR at https://github.com/dhall-lang/dhall-lang/pull/1026 to add the .dhall extension on each file